ssl/record/ssl3_record_tls13.c

```diff
@@ -21,7 +21,7 @@
  *   -1: if the record's AEAD-authenticator is invalid or, if sending,
  *       an internal error occurred.
  */
-int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int send)
+int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int sending)
```

Comment:
> but I was talking in a more general sense.

But we are looking at specific case. In general it's fine, it's this one specific case that is controversial. Indeed, a global variable shadowing function is an error and compiler does fail. Local variable shadowing global variable is not prohibited, but users might appreciate the warning, and compiler does issue one. But local variable shadowing function is *unlikely* to be appreciated, rather contrary, *and* this is effectively confirmed by the fact that newer compilers do *not* issue warning in such case.